src/main/java/de/metas/ui/web/pickingV2/productsToPick/ProductsToPickRowsRepository.java

```diff
@@ -81,11 +81,11 @@ public PickHURequest createPickHURequest(@NonNull final ProductsToPickRow row, f
 				.build();
 	}
 
-	public List<PickingCandidate> pick(@NonNull final PackageableRow packageableRow)
+	public List<PickingCandidate> createPickingCandidates(@NonNull final PackageableRow packageableRow)
 	{
 		final ProductsToPickRowsData productsToPickRowsData = createProductsToPickRowsData(packageableRow);
 		return productsToPickRowsData.getAllRows().stream()
-				.map(productsToPickRow -> pickingCandidateService.pickHU(createPickHURequest(productsToPickRow, false)))
+				.map(productsToPickRow -> pickingCandidateService.createAndSavePickingCandidates(createPickHURequest(productsToPickRow, false)))
```

Comment:
```suggestion
				.map(productsToPickRow -> pickingCandidateService.createAndSavePickingCandidates(createPickHURequest(productsToPickRow, false /*createPickingCandidatesOnly*/)))
```
pls add the parameter name for better readability